### PR TITLE
Adding new meetups in town.

### PR DESCRIPTION
**Pull request by WritingPanda**
Added:

- [San Antonio Code Newbies](https://www.meetup.com/San-Antonio-Code-Newbies/)
- [Lisk Javascript Workshop](https://www.meetup.com/LiskCommunityJavaScriptWorkshop/)
- [AI/ML Side Projects and Learning](https://www.meetup.com/Artificial-Intelligence-and-Machine-Learning-of-San-Antonio/)
- [San Antonio Women Developers](https://www.meetup.com/San-Antonio-Women-Developers/)
- [Code for San Antonio](https://www.meetup.com/Code-for-San-Antonio/)
- [San Antonio Linux Enthusiasts](https://www.meetup.com/San-Antonio-Linux-Enthusiasts/)

### CODE REVIEW

**Comment by WritingPanda** [comment]
Also updated the how-to doc on the site to make it more agnostic and use make.

File: content/articles/add_docs.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ Our site is a static site hosted somewhere out there on the Internet. Super myst
 
 The site is powered by Pelican, which is a static site generator written in Python. If you have Python on your machine and you want to see what your article looks like on your local machine, install Pelican by typing in your terminal 
 
-    pip install pipenv # if you don't have pipenv on your machine already
-    pipenv install
+    pip install -r requirements.txt
```
Comment (gtmanfred) [inline content/articles/add_docs.md]: we don't have a requirements.txt right now.

I would rather do that in a separate pr, and have more generic requirements based on what the application uses, instead of hard coding them to a specific release version.
